api/v1alpha1/bundle_webhook.go

```diff
@@ -76,6 +76,14 @@ func checkBundleSource(r *Bundle) error {
 		if r.Spec.Source.Git == nil {
 			return fmt.Errorf("bundle.spec.source.git must be set for source type \"git\"")
 		}
+		if r.Spec.Source.Git.Directory != "" {
+			if r.Spec.Source.Git.Directory[0] == '/' {
+				return fmt.Errorf("bundle.spec.source.git.directory can be \"/\"")
+			}
+			if len(r.Spec.Source.Git.Directory) > 2 && r.Spec.Source.Git.Directory[:3] == "../" {
```

Comment:
1 nit, 1 suggestion:
- nit: use `strings.HasPrefix` to simplify existing check.
- suggestion: we need to use `filepath.Clean` here as well 
    - `foo/../../` is the same as `../`. This code would pass that directory, but that should fail.
```suggestion
			if strings.HasPrefix(filepath.Clean(r.Spec.Source.Git.Directory), "../") {
```

Reply:
I'll take your suggestion.  Thanks!